### PR TITLE
Link Fix & Qemu Monitor Over Tellnet

### PR DESCRIPTION
**Pull request by maxtyson123**
Whilst using the PDF version of the book I noticed you could not access the website: https://jmnl.xyz/windowmanager/ due to a typo.

I also added information on how to access the monitor using Telnet as I found that to be easier to use with my setup and thought it may be helpful. 

(I use WSL but use qemu on the windows side, however my IDE creates the terminals in WSL so telnet helps bridge the two to access the monitor, additionally I use serial of studio so cant use monitor there)

### CODE REVIEW

**Review by DeanoBurrito — CHANGES_REQUESTED** [review]

File: 99_Appendices/F_Debugging.md [hunk 1]
```diff
@@ -212,19 +212,30 @@ then on another shell, on the same folder where we started the emulator launch t
...
 
-This will prompt with a shell similar to the following:
+
+Another method is to use telnet to start the monitor. This is handy for easier, cross-platform or remote use (albeit less secure).
+
+```bash
+qemu-system-i386 [..other params..] -monitor telnet::45454,server,nowait
+```
+
+This enables the monitor to listen on a specified port (ie, 45454). You can then connect to the QEMU monitor from another terminal or a remote machine (with networking setup) using Telnet:
```
Comment (dreamos82) [inline 99_Appendices/F_Debugging.md:222]: i see telnet here with capital T, and in the previous sentence with lowercase t, maybe we want to use the same capitalization on both? Or is there a reason?

**Review by dreamos82 — COMMENTED** [review]
Just a minor comment, anyway feel free to add yourself in the Acknowledgments appendix! :)

File: 99_Appendices/F_Debugging.md (hunk 1 above)
Comment (maxtyson123) [reply 99_Appendices/F_Debugging.md:222]: Thank you, also fixed some to do with Qemu, didn't know if there were more

**Review by DeanoBurrito — APPROVED** [review]

**Comment by DeanoBurrito** [comment]
lgtm, thanks for the fixes!

**Pull request merged**